mantidproject/mantid
# Enable rebin options in frequency domain analysis

**Description of work.**

This enables the use raw option in the FDA tab. It also exposes the new interface to the users and adds documentation and release notes.

**Report to:** Anthony 

**To test:**

* Open FDA analysis load some data and set a Binning on the home tab.
* Switch to the transform tab, and check the work spaces available from the drop down menu. This should include all of the groups and pairs as well as a PhaseQuad. 
* Untick the use raw box and check again the available workspaces should now be the rebinned ones.

* Click on the question mark in the lower left hand corner of the interface and check a help page comes up (You will need to have built the docs-qthelp target).
* Read through the documentation and check it makes sense and is fit for purpose. 
* Check that Max Ent and Phase Quad work for multiperiod data. 
* Check that the old Frequency domain analysis still works, (Max Ent for multiperiod data does not work in the old version but this seems to be the case on master as well.)



Fixes #24748  




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
